src/legacy/core_plugins/kibana/migrations.js

```diff
@@ -311,16 +316,71 @@ function migrateFiltersAggQuery(doc) {
   return doc;
 }
 
-const executeMigrations720 = flow(migratePercentileRankAggregation, migrateDateHistogramAggregation);
-const executeMigrations730 = flow(migrateGaugeVerticalSplitToAlignment, transformFilterStringToQueryObject, migrateFiltersAggQuery);
+function replaceMovAvgToMovFn(doc) {
+  const visStateJSON = get(doc, 'attributes.visState');
+  let visState;
+
+  if (visStateJSON) {
+    try {
+      visState = JSON.parse(visStateJSON);
+    } catch (e) {
+      // Let it go, the data is invalid and we'll leave it as is
+    }
```

Comment:
Just one last minor change: could you please extend the try catch to the full length of the change as in `migrateGaugeVerticalSplitToAlignment` and add logging to log exceptions?

Reply:
It makes sense. Done. Thanks.